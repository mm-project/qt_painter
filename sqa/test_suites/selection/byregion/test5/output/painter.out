QWidget::paintEngine: Should no longer be called
QPainter::begin: Paint device returned engine == 0, type: 1
incmdCreateObjRectangle
dicmdCanvasMouseMove -point (205,23) 
dicmdCanvasMouseMove -point (205,23) 
dicmdCanvasMouseMove -point (182,43) 
dicmdCanvasMouseMove -point (182,43) 
dicmdCanvasMouseMove -point (163,58) 
dicmdCanvasMouseMove -point (163,58) 
dicmdCanvasMouseMove -point (148,69) 
dicmdCanvasMouseMove -point (148,69) 
dicmdCanvasMouseMove -point (141,75) 
dicmdCanvasMouseMove -point (141,75) 
dicmdCanvasMouseMove -point (138,78) 
dicmdCanvasMouseMove -point (138,78) 
dicmdCanvasMouseMove -point (137,79) 
dicmdCanvasMouseMove -point (137,79) 
dicmdCanvasMouseMove -point (136,79) 
dicmdCanvasMouseMove -point (136,79) 
dicmdCanvasMouseMove -point (135,79) 
dicmdCanvasMouseMove -point (135,79) 
dicmdCanvasMouseMove -point (134,79) 
dicmdCanvasMouseMove -point (134,79) 
dicmdCanvasMouseMove -point (133,79) 
dicmdCanvasMouseMove -point (133,79) 
dicmdCanvasMouseMove -point (132,78) 
dicmdCanvasMouseMove -point (132,78) 
dicmdCanvasMouseMove -point (131,78) 
dicmdCanvasMouseMove -point (131,78) 
dicmdCanvasMouseMove -point (131,77) 
dicmdCanvasMouseMove -point (131,77) 
dicmdCanvasMouseClick -point (131,77) 
dicmdCanvasMouseMove -point (131,77) 
dicmdCanvasMouseClick -point (131,77) 
dicmdCanvasMouseMove -point (135,80) 
dicmdCanvasMouseMove -point (135,80) 
dicmdCanvasMouseMove -point (141,86) 
dicmdCanvasMouseMove -point (141,86) 
dicmdCanvasMouseMove -point (154,97) 
dicmdCanvasMouseMove -point (154,97) 
dicmdCanvasMouseMove -point (169,110) 
dicmdCanvasMouseMove -point (169,110) 
dicmdCanvasMouseMove -point (188,128) 
dicmdCanvasMouseMove -point (188,128) 
dicmdCanvasMouseMove -point (201,140) 
dicmdCanvasMouseMove -point (201,140) 
dicmdCanvasMouseMove -point (211,148) 
dicmdCanvasMouseMove -point (211,148) 
dicmdCanvasMouseMove -point (220,155) 
dicmdCanvasMouseMove -point (220,155) 
dicmdCanvasMouseMove -point (229,163) 
dicmdCanvasMouseMove -point (229,163) 
dicmdCanvasMouseMove -point (234,168) 
dicmdCanvasMouseMove -point (234,168) 
dicmdCanvasMouseMove -point (238,171) 
dicmdCanvasMouseMove -point (238,171) 
dicmdCanvasMouseMove -point (240,172) 
dicmdCanvasMouseMove -point (240,172) 
dicmdCanvasMouseMove -point (240,173) 
dicmdCanvasMouseMove -point (240,173) 
dicmdCanvasMouseMove -point (241,173) 
dicmdCanvasMouseMove -point (241,173) 
dicmdCanvasMouseMove -point (242,173) 
dicmdCanvasMouseMove -point (242,173) 
dicmdCanvasMouseClick -point (242,173) 
dicmdCanvasMouseMove -point (242,173) 
dicmdCanvasMouseClick -point (242,173) 
#o
#o command not found
dicmdCanvasMouseMove -point (242,172) 
dicmdCreateObjRectangle -points {(131,77);(242,172)} 
dicmdCanvasMouseMove -point (242,172) 
dicmdCanvasMouseMove -point (242,169) 
dicmdCanvasMouseMove -point (242,169) 
dicmdCanvasMouseMove -point (242,165) 
dicmdCanvasMouseMove -point (242,165) 
dicmdCanvasMouseMove -point (241,161) 
dicmdCanvasMouseMove -point (241,161) 
dicmdCanvasMouseMove -point (239,155) 
dicmdCanvasMouseMove -point (239,155) 
dicmdCanvasMouseMove -point (235,150) 
dicmdCanvasMouseMove -point (235,150) 
dicmdCanvasMouseMove -point (232,147) 
dicmdCanvasMouseMove -point (232,147) 
dicmdCanvasMouseMove -point (228,145) 
dicmdCanvasMouseMove -point (228,145) 
dicmdCanvasMouseMove -point (225,143) 
dicmdCanvasMouseMove -point (225,143) 
dicmdCanvasMouseMove -point (223,143) 
dicmdCanvasMouseMove -point (223,143) 
dicmdCanvasMouseMove -point (221,142) 
dicmdCanvasMouseMove -point (221,142) 
dicmdCanvasMouseMove -point (220,142) 
dicmdCanvasMouseMove -point (220,142) 
dicmdCanvasMouseMove -point (219,142) 
dicmdCanvasMouseMove -point (219,142) 
dicmdCanvasMouseMove -point (218,142) 
dicmdCanvasMouseMove -point (218,142) 
dicmdCanvasMouseMove -point (217,142) 
dicmdCanvasMouseMove -point (217,142) 
dicmdCanvasMouseClick -point (217,142) 
dicmdCanvasMouseMove -point (217,142) 
dicmdCanvasMouseClick -point (217,142) 
dicmdCanvasMouseMove -point (218,147) 
dicmdCanvasMouseMove -point (218,147) 
dicmdCanvasMouseMove -point (227,158) 
dicmdCanvasMouseMove -point (227,158) 
dicmdCanvasMouseMove -point (251,183) 
dicmdCanvasMouseMove -point (251,183) 
dicmdCanvasMouseMove -point (276,210) 
dicmdCanvasMouseMove -point (276,210) 
dicmdCanvasMouseMove -point (296,234) 
dicmdCanvasMouseMove -point (296,234) 
dicmdCanvasMouseMove -point (313,247) 
dicmdCanvasMouseMove -point (313,247) 
dicmdCanvasMouseMove -point (326,254) 
dicmdCanvasMouseMove -point (326,254) 
dicmdCanvasMouseMove -point (335,258) 
dicmdCanvasMouseMove -point (335,258) 
dicmdCanvasMouseMove -point (340,260) 
dicmdCanvasMouseMove -point (340,260) 
dicmdCanvasMouseMove -point (342,261) 
dicmdCanvasMouseMove -point (342,261) 
dicmdCanvasMouseMove -point (343,262) 
dicmdCanvasMouseMove -point (343,262) 
dicmdCanvasMouseMove -point (344,262) 
dicmdCanvasMouseMove -point (344,262) 
dicmdCanvasMouseClick -point (344,262) 
dicmdCanvasMouseMove -point (344,262) 
dicmdCanvasMouseClick -point (344,262) 
#o
#o command not found
dicmdCanvasMouseMove -point (342,262) 
dicmdCreateObjRectangle -points {(217,142);(342,262)} 
dicmdCanvasMouseMove -point (342,262) 
dicmdCanvasMouseMove -point (339,260) 
dicmdCanvasMouseMove -point (339,260) 
dicmdCanvasMouseMove -point (335,256) 
dicmdCanvasMouseMove -point (335,256) 
dicmdCanvasMouseMove -point (330,249) 
dicmdCanvasMouseMove -point (330,249) 
dicmdCanvasMouseMove -point (323,244) 
dicmdCanvasMouseMove -point (323,244) 
dicmdCanvasMouseMove -point (319,241) 
dicmdCanvasMouseMove -point (319,241) 
dicmdCanvasMouseMove -point (315,239) 
dicmdCanvasMouseMove -point (315,239) 
dicmdCanvasMouseMove -point (312,238) 
dicmdCanvasMouseMove -point (312,238) 
dicmdCanvasMouseMove -point (311,237) 
dicmdCanvasMouseMove -point (311,237) 
dicmdCanvasMouseMove -point (310,236) 
dicmdCanvasMouseMove -point (310,236) 
dicmdCanvasMouseMove -point (309,236) 
dicmdCanvasMouseMove -point (309,236) 
dicmdCanvasMouseMove -point (309,235) 
dicmdCanvasMouseMove -point (309,235) 
dicmdCanvasMouseMove -point (310,237) 
dicmdCanvasMouseMove -point (310,237) 
dicmdCanvasMouseMove -point (313,240) 
dicmdCanvasMouseMove -point (313,240) 
dicmdCanvasMouseMove -point (316,245) 
dicmdCanvasMouseMove -point (316,245) 
dicmdCanvasMouseClick -point (316,245) 
dicmdCanvasMouseMove -point (316,245) 
dicmdCanvasMouseClick -point (316,245) 
dicmdCanvasMouseMove -point (324,251) 
dicmdCanvasMouseMove -point (324,251) 
dicmdCanvasMouseMove -point (344,262) 
dicmdCanvasMouseMove -point (344,262) 
dicmdCanvasMouseMove -point (368,277) 
dicmdCanvasMouseMove -point (368,277) 
dicmdCanvasMouseMove -point (402,295) 
dicmdCanvasMouseMove -point (402,295) 
dicmdCanvasMouseMove -point (423,307) 
dicmdCanvasMouseMove -point (423,307) 
dicmdCanvasMouseMove -point (438,316) 
dicmdCanvasMouseMove -point (438,316) 
dicmdCanvasMouseMove -point (447,321) 
dicmdCanvasMouseMove -point (447,321) 
dicmdCanvasMouseMove -point (450,323) 
dicmdCanvasMouseMove -point (450,323) 
dicmdCanvasMouseMove -point (452,324) 
dicmdCanvasMouseMove -point (452,324) 
dicmdCanvasMouseMove -point (453,324) 
dicmdCanvasMouseMove -point (453,324) 
dicmdCanvasMouseMove -point (453,325) 
dicmdCanvasMouseMove -point (453,325) 
dicmdCanvasMouseMove -point (454,325) 
dicmdCanvasMouseMove -point (454,325) 
dicmdCanvasMouseClick -point (454,325) 
dicmdCanvasMouseMove -point (454,325) 
dicmdCanvasMouseClick -point (454,325) 
#o
#o command not found
dicmdCanvasMouseMove -point (453,324) 
dicmdCreateObjRectangle -points {(316,245);(453,324)} 
dicmdCanvasMouseMove -point (453,324) 
dicmdCanvasMouseMove -point (452,323) 
dicmdCanvasMouseMove -point (452,323) 
dicmdCanvasMouseMove -point (451,321) 
dicmdCanvasMouseMove -point (451,321) 
dicmdCanvasMouseMove -point (449,319) 
dicmdCanvasMouseMove -point (449,319) 
dicmdCanvasMouseMove -point (448,317) 
dicmdCanvasMouseMove -point (448,317) 
dicmdCanvasMouseMove -point (447,314) 
dicmdCanvasMouseMove -point (447,314) 
dicmdCanvasMouseMove -point (445,312) 
dicmdCanvasMouseMove -point (445,312) 
dicmdCanvasMouseMove -point (444,310) 
dicmdCanvasMouseMove -point (444,310) 
dicmdCanvasMouseMove -point (443,309) 
dicmdCanvasMouseMove -point (443,309) 
dicmdCanvasMouseMove -point (442,307) 
dicmdCanvasMouseMove -point (442,307) 
dicmdCanvasMouseMove -point (441,305) 
dicmdCanvasMouseMove -point (441,305) 
dicmdCanvasMouseMove -point (439,303) 
dicmdCanvasMouseMove -point (439,303) 
dicmdCanvasMouseMove -point (438,302) 
dicmdCanvasMouseMove -point (438,302) 
dicmdCanvasMouseMove -point (437,301) 
dicmdCanvasMouseMove -point (437,301) 
dicmdCanvasMouseMove -point (437,300) 
dicmdCanvasMouseMove -point (437,300) 
dicmdCanvasMouseMove -point (436,299) 
dicmdCanvasMouseMove -point (436,299) 
dicmdCanvasMouseMove -point (435,298) 
dicmdCanvasMouseMove -point (435,298) 
dicmdCanvasMouseMove -point (435,297) 
dicmdCanvasMouseMove -point (435,297) 
dicmdCanvasMouseMove -point (434,297) 
dicmdCanvasMouseMove -point (434,297) 
dicmdCanvasMouseMove -point (434,296) 
dicmdCanvasMouseMove -point (434,296) 
dicmdCanvasMouseMove -point (433,296) 
dicmdCanvasMouseMove -point (433,296) 
dicmdCanvasMouseMove -point (432,296) 
dicmdCanvasMouseMove -point (432,296) 
dicmdCanvasMouseMove -point (432,297) 
dicmdCanvasMouseMove -point (432,297) 
dicmdQaCompareDesign 
comparision->Design:PASS Design_Compare_0.txt Design_Compare_0.txt.golden
#t
#t command not found
dicmdCanvasMouseMove -point (432,298) 
dicmdCanvasMouseMove -point (432,298) 
dicmdCanvasMouseMove -point (432,297) 
dicmdCanvasMouseMove -point (432,297) 
dicmdCanvasMouseMove -point (432,296) 
dicmdCanvasMouseMove -point (432,296) 
dicmdCanvasMouseMove -point (432,295) 
dicmdCanvasMouseMove -point (432,295) 
dicmdCanvasMouseMove -point (432,293) 
dicmdCanvasMouseMove -point (432,293) 
dicmdCanvasMouseMove -point (438,285) 
dicmdCanvasMouseMove -point (438,285) 
dicmdCanvasMouseMove -point (469,256) 
dicmdCanvasMouseMove -point (469,256) 
dicmdCanvasMouseMove -point (523,207) 
dicmdCanvasMouseMove -point (523,207) 
dicmdCanvasMouseMove -point (590,166) 
dicmdCanvasMouseMove -point (590,166) 
dicmdCanvasMouseMove -point (646,136) 
dicmdCanvasMouseMove -point (646,136) 
dicmdCanvasMouseMove -point (711,102) 
dicmdCanvasMouseMove -point (711,102) 
dicmdCanvasMouseMove -point (752,88) 
dicmdCanvasMouseMove -point (752,88) 
dicmdCanvasMouseMove -point (793,74) 
dicmdCanvasMouseMove -point (793,74) 
dicmdCanvasMouseMove -point (825,63) 
dicmdCanvasMouseMove -point (825,63) 
dicmdCanvasMouseMove -point (853,51) 
dicmdCanvasMouseMove -point (853,51) 
dicmdCanvasMouseMove -point (869,45) 
dicmdCanvasMouseMove -point (869,45) 
dicmdCanvasMouseMove -point (866,48) 
dicmdCanvasMouseMove -point (866,48) 
dicmdCanvasMouseMove -point (863,49) 
dicmdCanvasMouseMove -point (863,49) 
dicmdCanvasMouseMove -point (859,52) 
dicmdCanvasMouseMove -point (859,52) 
dicmdCanvasMouseMove -point (852,57) 
dicmdCanvasMouseMove -point (852,57) 
dicmdCanvasMouseMove -point (841,64) 
dicmdCanvasMouseMove -point (841,64) 
dicmdCanvasMouseMove -point (833,70) 
dicmdCanvasMouseMove -point (833,70) 
dicmdCanvasMouseMove -point (826,75) 
dicmdCanvasMouseMove -point (826,75) 
dicmdCanvasMouseMove -point (819,80) 
dicmdCanvasMouseMove -point (819,80) 
dicmdCanvasMouseMove -point (814,83) 
dicmdCanvasMouseMove -point (814,83) 
dicmdCanvasMouseMove -point (811,86) 
dicmdCanvasMouseMove -point (811,86) 
dicmdCanvasMouseMove -point (810,86) 
dicmdCanvasMouseMove -point (810,86) 
dicmdCanvasMouseMove -point (809,87) 
dicmdCanvasMouseMove -point (809,87) 
dicmdCanvasMouseMove -point (808,87) 
dicmdCanvasMouseMove -point (808,87) 
dicmdCanvasMouseMove -point (807,87) 
dicmdCanvasMouseMove -point (807,87) 
dicmdCanvasMouseMove -point (806,87) 
dicmdCanvasMouseMove -point (806,87) 
dicmdCanvasMouseMove -point (805,87) 
dicmdCanvasMouseMove -point (805,87) 
dicmdQaCompareSelection 
comparision->Selection:PASS Selection_Compare_0.txt Selection_Compare_0.txt.golden
#t
#t command not found
dicmdCanvasMouseMove -point (804,87) 
dicmdCanvasMouseMove -point (804,87) 
dicmdCanvasMouseMove -point (807,87) 
dicmdCanvasMouseMove -point (807,87) 
dicmdCanvasMouseMove -point (819,86) 
dicmdCanvasMouseMove -point (819,86) 
dicmdCanvasMouseMove -point (875,65) 
dicmdCanvasMouseMove -point (875,65) 
dicmdCanvasMouseMove -point (960,41) 
dicmdCanvasMouseMove -point (960,41) 
dicmdCanvasMouseMove -point (1036,26) 
dicmdCanvasMouseMove -point (1036,26) 
dicmdCanvasMouseMove -point (1085,17) 
dicmdCanvasMouseMove -point (1085,17) 
dicmdCanvasMouseMove -point (1117,9) 
dicmdCanvasMouseMove -point (1117,9) 
dicmdCanvasMouseMove -point (1139,4) 
dicmdCanvasMouseMove -point (1139,4) 
dicmdCanvasMouseMove -point (1149,0) 
dicmdCanvasMouseMove -point (1149,0) 
dicmdQaToolExit 
