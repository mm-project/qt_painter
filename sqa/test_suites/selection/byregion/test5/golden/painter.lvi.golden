dicmdCreateObjRectangle -brush 1 -color #000000 -fill 9 -points {(131,77);(242,173)} 
dicmdCreateObjRectangle -brush 1 -color #000000 -fill 9 -points {(217,142);(344,262)} 
dicmdCreateObjRectangle -brush 1 -color #000000 -fill 9 -points {(316,245);(454,325)} 
